php/actions/get_torrent_files.php

```diff
@@ -83,11 +83,18 @@
     // скачивание торрент-файлов
     $download = new TorrentDownload($cfg['forum_address']);
 
-    Log::append(
-        $replace_passkey
-            ? 'Выполняется скачивание торрент-файлов с заменой Passkey...'
-            : 'Выполняется скачивание торрент-файлов...'
+    $log_string = sprintf(
+        "Выполняется скачивание торрент-файлов (%d шт), трекеры %s. ",
+        count($topicHashes['topic_hashes']),
+        $cfg['tor_for_user'] ? 'пользовательские' : 'хранительские'
     );
+    if ($replace_passkey) {
+        $log_string .=
+            !empty($cfg['user_passkey'])
+                ? 'Замена Passkey: ' .$cfg['user_passkey']
+                : 'Passkey пуст. Использован стандартный трекер.';
```

Comment:
```suggestion
                : 'Passkey пуст.';
```

The tracker type is already mentioned on the line 89.

Reply:
done